### DISCUSSION TITLE
Paper writing - Description of abstention metrics

### DISCUSSION THREAD

**Discussion by hgissbkh** [embeddings-benchmark/mteb#895] [General]
Hey @KennethEnevoldsen 
As discussed, here is a discussion to talk about the description of abstention metrics I wrote in Appendix E
Let me know what you think about it and if the tone seems right to you!
Ref: [https://github.com/embeddings-benchmark/mteb/issues/784](https://github.com/embeddings-benchmark/mteb/issues/784)

**Comment by KennethEnevoldsen** [comment]
@hgissbkh I have renamed the section to new metrics as I believe that is what it contains. I have also added a few comments - let me know once you have looked at it then we can wrap up the section.

**Reply by hgissbkh** [reply]
Just accepted almost all you suggestions
But I'm not sure to understand how you want me to rephrase the "compute upper-bound" step

**Reply by hgissbkh** [reply]
Was also wondering if this writing contribution was eligible for some additional points. I saw there was a dedicated section in the leaderboard :)

**Reply by KennethEnevoldsen** [reply]
I think it is fine as it is. Thanks!  You can create a PR with 3 points (I believe the section is basically finished). If you wish you might also want to see if we have added additional metrics since MTEB and add those in as well.
